pkg/asset/installconfig/azure/validation.go

```diff
@@ -104,10 +112,10 @@ func ValidateInstanceType(client API, fieldPath *field.Path, region, instanceTyp
 		}
 		if cpus < float64(req.minimumVCpus) {
 			errMsg := fmt.Sprintf("instance type does not meet minimum resource requirements of %d vCPUsAvailable", req.minimumVCpus)
-			allErrs = append(allErrs, field.Invalid(fieldPath.Child("type"), instanceType, errMsg))
+			allErrs = append(allErrs, field.Invalid(fieldPath, instanceType, errMsg))
 		}
 	} else {
-		allErrs = append(allErrs, field.Invalid(fieldPath.Child("type"), instanceType, "capability not found: vCPUsAvailable"))
+		allErrs = append(allErrs, field.Invalid(fieldPath, instanceType, "capability not found: vCPUsAvailable"))
```

Comment:
So the previous behavior did not fail here when checking memory or vcpus and they couldn't be found? Let's just log a WARNING or DEBUG message instead of failing if that's more consistent with the previous behavior...